backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -83,33 +116,43 @@ object BackendSettings extends BackendSettingsApi {
           "ArrayType as Value type in MapType"
       }
       for (unsupportedDataType <- unsupportedDataTypes) {
-        // scalastyle:off println
-        println(
+        logWarning(
           s"Validation failed for ${this.getClass.toString}" +
             s" due to: data type $unsupportedDataType. in file schema. ")
-        // scalastyle:on println
       }
       unsupportedDataTypes.isEmpty
     }
 
+    // check if supported file system
+    val unsupportedPath = rootPaths.find(!isSupportedFileSystem(_))
```

Comment:
Do we need to verify all the root paths? Is it a usual case for files belonging to different systems?

Reply:
> Do we need to verify all the root paths? Is it a usual case for files belonging to different systems?

Maybe first is enough. I think the rootPaths of one datasource are mostly the same filesystem.

Reply:
@PHILO-HE What do you think?

Reply:
Maybe, multiple kinds of file system are not commonly used in scan. We may just validate the first one. If doing so, let's leave some comment to clarify.

Reply:
Fixed, thanks